package.json

```diff
@@ -57,7 +58,6 @@
     "qunit": "^2.16.0",
     "qunit-dom": "^1.6.0"
   },
-  "peerDependencies": {},
```

Comment:
Also add konva versions `>=7.1.4` (the earliest version I tested with) into the peer dependencies.
So you can provide your own konva version in the project consuming this addon.
```
"peerDependencies": {
  "konva": ">=7.1.4"
}
```